src/layouts/shared/NetworkErrorBoundary/NetworkErrorBoundary.jsx

```diff
@@ -209,8 +217,8 @@ class NetworkErrorBoundary extends Component {
           src={illustration}
         />
         <h1 className="mt-6 text-2xl">{title}</h1>
-        {description && <p className="mt-6">{description(data)}</p>}
-        <NetworkErrorMessage />
+        {description && <p className="mt-2">{description(data)}</p>}
+        {showDocs && <NetworkErrorMessage />}
```

Comment:
Can you swap this for a ternary 👀

Reply:
We do now have a [notion doc](https://www.notion.so/sentry/Use-ternaries-rather-than-in-JSX-5ccc5225e4fb468a8165071e891afaac) explaining the reasoning behind favouring the ternaries over double ampersands.

Reply:
oh yeah no prob, lemme swap the others too